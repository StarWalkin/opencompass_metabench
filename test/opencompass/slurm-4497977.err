  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:12<00:00, 12.31s/it]100%|██████████| 1/1 [00:12<00:00, 12.31s/it]
  0%|          | 0/1 [00:00<?, ?it/s]                                       0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:10<00:00, 10.09s/it]100%|██████████| 1/1 [00:10<00:00, 10.09s/it]
