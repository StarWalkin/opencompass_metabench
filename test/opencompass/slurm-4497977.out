/u/zzhang27/.conda/envs/opencompass/bin/python3
08/17 04:35:21 - OpenCompass - INFO - Loading gsm8k_gen: configs/datasets/gsm8k/gsm8k_gen.py
08/17 04:35:21 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/17 04:35:21 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/17 04:35:21 - OpenCompass - INFO - Current exp folder: outputs/default/20240817_043521
08/17 04:35:21 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/17 04:35:21 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLInfer[gemma2-9b-it-hf/gsm8k] on GPU 0
08/17 04:35:33 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/local.py - _launch - 228 - task OpenICLInfer[gemma2-9b-it-hf/gsm8k] fail, see
outputs/default/20240817_043521/logs/infer/gemma2-9b-it-hf/gsm8k.out
08/17 04:35:33 - OpenCompass - ERROR - /scratch/bcwg/Metabench/repo/test/opencompass/opencompass/runners/base.py - summarize - 64 - OpenICLInfer[gemma2-9b-it-hf/gsm8k] failed with code 1
08/17 04:35:33 - OpenCompass - INFO - Partitioned into 1 tasks.
launch OpenICLEval[gemma2-9b-it-hf/gsm8k] on CPU 
dataset    version    metric    mode    gemma2-9b-it-hf
---------  ---------  --------  ------  -----------------
gsm8k      -          -         -       -
08/17 04:35:43 - OpenCompass - INFO - write summary to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_043521/summary/summary_20240817_043521.txt
08/17 04:35:43 - OpenCompass - INFO - write csv to /scratch/bcwg/Metabench/repo/test/opencompass/outputs/default/20240817_043521/summary/summary_20240817_043521.csv
All processes have been started and their PIDs stored in process_ids_gpt4o.txt.
All processes have completed.
